__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require("supertest");
+const db = require("../db/connection.js");
+const { app } = require("../app.js")
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+
+describe('testing for get /api/topics', () => {
+    test('returns status code 200', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    })
+    test('returns an array of topics with correct properties', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((result) => {
+            const {topics} = result.body
+            expect(topics.length).toBe(3)
+            topics.forEach((topic) => {
+                expect(typeof topic.slug).toBe('string')
+                expect(typeof topic.description).toBe('string') 
+            })
```

Comment:
Nice tests, you might want to consider using matchers like toMatchObject to achieve the same thing more succinctly